src/content/docs/logs/ui-data/query-time-parsing.mdx

```diff
@@ -0,0 +1,412 @@
+---
+title: Query time parsing in logs
+tags:
+  - Logs
+  - Log management
+  - UI and data
+metaDescription: How New Relic uses parsing and how to send customized log data.
+freshnessValidatedDate: 2024-05-15 
+---
+
+
+
+
+import logsLogsTableEntryPoint from 'images/logs_screenshot-full_logs-table-entry-point.webp'
+
+import logsLogsDetailEntryPoint from 'images/logs_screenshot-full_logs-detail-entry-point.webp'
+
+import logsOpenQueryTimeParsingEditor from 'images/logs_screenshot-full_open-query-time-parsing-editor.webp'
+
+import logsExtractValuesinParserRule from 'images/logs_screenshot-crop_extract-values-in-parser-rule.webp'
+
+import logsEditorShowsParsedDataPreview from 'images/logs_screenshot-crop_editor-shows-parsed-data-preview.webp'
+
+import logsNrqlBehindQueryRule from 'images/logs_screenshot-crop_nrql-behind-query-rule.webp'
+
+import logsDeleteQueryTimeRule from 'images/logs_screenshot-crop_delete-query-time-rule.webp'
+
+import logsViewResultsinLogDetailsView from 'images/logs_screenshot-full_view-results-in-log-details-view.webp'
+
+import logsLogTableResults from 'images/logs_screenshot-full_log-table-results.webp'
+
+import logsRuleManager from 'images/logs_screenshot-crop_rule-manager.webp'
+
+import logsQueryTimeSearchBar from 'images/logs_screenshot-full_query-time-search-bar.webp'
+
+import logsAlertQueryBuilder from 'images/logs_screenshot-crop_alert-query-builder.webp'
+
+import logsQueryBehindParsingRule from 'images/logs_screenshot-crop_query-behind-parsing-rule.webp'
+
+import logsParsingAttributesView from 'images/logs_screenshot-crop_parsing-attributes-view.webp'
+
+
+
+
+
+Query time parsing lets you parse your logs directly in the UI without needing to write complex regular expressions or grok patterns. Results are shown instantly since parsing is performed at query time. You can use query time parsing to temporarily extract values from your logs and quickly perform a query on these variables.
```

Comment:
The word Grok here feels out of place. Primarily because it feels like slang or a colloquial term, but also I think it gets sticky with New Relic Grok. I'd love it if you could get more specific, which might be hard to do for the first sentence. But why does query time parsing mean you don't have to use regular expressions?

Reply:
I revisited this- I think I was misunderstanding Grok. We can sync about this later. I still think there needs to be a greater call to action at the beginning. I think a use case or an example would be super helpful here. It would also be in line with what our customers request. It feels like I don't have anything to ground myself within. Why is this a special new feature? Why use this rather than another method other than to save yourself time?

Reply:
I've revised the introduction.